recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -71,6 +71,21 @@ def _min_cppstd(self):
             return 14
         return 11
 
+    @property
+    def _default_cppstd(self):
+        if self.options.with_stl == "default":
+            if self.settings.compiler.cppstd:
```

Comment:
Might be better to use `self.settings.get_safe(“compiler.cppstd”)`?

Reply:
Ok, then we should also use it there?

https://github.com/conan-io/conan-center-index/blob/7848c1c8b0ab4097819cf4f93e901c01d66fddd8/recipes/opentelemetry-cpp/all/conanfile.py#L147

https://github.com/conan-io/conan-center-index/blob/7848c1c8b0ab4097819cf4f93e901c01d66fddd8/recipes/opentelemetry-cpp/all/conanfile.py#L230